src/components/Preview/Preview.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Preview.css';
+
+export default function Preview({ head, tops, bottoms }) {
+  return (
+    <div className="preview">
+      <div className="character">
+        <img src={`${process.env.PUBLIC_URL}/icons/${head}.png`} className="character head" />
```

Comment:
If you want to read up on why you should use the public folder when you need to dynamically reference image paths you can check out an explanation for it in the create-react-app docs [here](https://create-react-app.dev/docs/using-the-public-folder)